skops/io/_scikeras.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import annotations
+
+import io
+import os
+import tempfile
+from typing import Sequence, Type
+
+import tensorflow as tf
+from scikeras.wrappers import KerasClassifier, KerasRegressor
```

Comment:
this is making `scikeras` a hard dependency in effect, while it should be a soft dependency (we only care about it if user has it installed). Also, we shouldn't even try to import anything from `scikeras` if the user is not dealing with a scikeras model.

Reply:
Fixed this by adding a try except around the import, following how `_quantile_forest.py` handles importing